src/main/java/at/porscheinformatik/seleniumcomponents/clarity/ClaritySelectContainerComponent.java

```diff
@@ -34,6 +35,13 @@ public String getSelectedValue()
         return select.getSelectedOption().getValue();
     }
 
+    public String getSelectedValueWithoutIndex()
+    {
+        String value = getSelectedValue();
+
+        return value.substring(value.indexOf(": ") + 2);
```

Comment:
I'm pretty sure this does not work under all circumstances. This is the format angular uses in the ngValue directive. If you set the value directly it looks like this:

![image](https://user-images.githubusercontent.com/2047555/189069576-dac4a4f3-945e-469f-bb87-bbbc35c80674.png)

Reply:
Ah, I see. That's unfortunate. I'll remove the method.